src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardView.swift

```diff
@@ -82,6 +84,13 @@ class HomeStatisticsCardView: UIView {
 
 		primaryTrendImageView.layer.cornerRadius = primaryTrendImageView.bounds.width / 2
 		secondaryTrendImageView.layer.cornerRadius = secondaryTrendImageView.bounds.width / 2
+
+		// delete button/circle
+		let tap = UITapGestureRecognizer(target: self, action: #selector(onDeleteTapped(_:)))
+		deletionIndicator.addGestureRecognizer(tap)
+		deletionIndicator.center = CGPoint(x: 10, y: 6) // magic number because of a very static design
```

Comment:
Would it make sense to use a `UIButton` with highlighting?

Reply:
It's just a small circle. Buttons come with a tons of functionality we don't use here.

Reply:
But we could use the free highlighting and VoiceOver support :)